equinix/fabric_connection_schema.go

```diff
@@ -453,6 +453,7 @@ func createRedundancySch() map[string]*schema.Schema {
 		"group": {
 			Type:        schema.TypeString,
 			Computed:    true,
+			Optional:    true,
```

Comment:
The docs will need to be updated with this change: https://github.com/equinix/terraform-provider-equinix/blob/main/docs/resources/equinix_fabric_connection.md#nested-schema-for-a_sideaccess_pointportredundancy

Reply:
Updates pending PR Approval by @manu-equinix : https://github.com/manu-equinix/terraform-provider-equinix/pull/14

Reply:
@displague @thogarty This has been resolved(merged). Please re-approve.

Reply:
@manu-equinix approval given 👍